apache2_changehat-close-button-seleted-20190510.json

```diff
@@ -0,0 +1,15 @@
+{
+  "area": [
+    {
+      "xpos": 639,
+      "ypos": 440,
+      "width": 113,
+      "height": 36,
+      "type": "match"
+    }
+  ],
+  "properties": [],
+  "tags": [
+    "close-button-seleted"

```

Comment:
Typo: should be "selected". Please crosscheck with the test code though :)

Reply:
Done! Thanks for your review :)

Reply:
Welcome back @okurz :)